lib/filters/css.js

```diff
@@ -3,30 +3,28 @@
 const CleanCSS = require('clean-css');
 const minimatch = require('minimatch');
 const Promise = require('bluebird');
-let enable;
```

Comment:
Hey,

Any specific reason to remove this from global scope ?

Reply:
Hey, same as for #24 
once a value is cached, the config value is ignored so it's impossible to test it for both `true` and `false` scenario. 

If you prefer to keep it global, we need to make sure that the config value takes precedence

Reply:
OK makes sense for now.